Makefile

```
@@ -0,0 +1,162 @@
+.DEFAULT_GOAL := build
+
+clean:
```

```suggestion
.PHONY: clean
clean:
```
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html